templates/directives/full-date.html

```diff
@@ -0,0 +1,6 @@
+<div ng-if-start="date" class="relative-date-content">
```

Comment:
If I'm reading this correctly this behaves the same as just `ng-if="date"` - is that right?